include/openPMD/ChunkInfo.hpp

```diff
@@ -72,5 +77,27 @@ struct WrittenChunkInfo : ChunkInfo
     bool operator==(WrittenChunkInfo const &other) const;
 };
 
+// !< @todo Also add a ChunkTable for ReadChunkInfo or sth like that
```

Comment:
Open todo?

Reply:
I'll remove the todo as I don't really see an immediate use case for that atm.